src/config/module-context.xml

```diff
@@ -10,4 +10,6 @@
             </list>
         </property>
     </bean>
+
+    <bean class="eu.xenit.apix.SpringConfiguration"/>
```

Comment:
Line responsible for making sure that Alfresco adds the beans from the `"eu.xenit.apix.alfresco"` (sub-)package(s) in the main application context.